### ISSUE TITLE
[HELP] Sorry, I have a question.

### ISSUE THREAD

**Issue by ZijunSong** [PKU-RL/PTGM#6]
According to the description in your paper (section 4.2), training the goal prior model seems to be a supervised learning process to find the action index. However, in the code (`goal_prior/train.py`), training the goal prior model seems to be a reinforcement learning process, where the loss is calculated after the action is obtained and the label is compared, and the appropriate action is obtained through training. I am a little confused.

**Comment by YHQpkueecs** [comment]
Hi, the training code you mentioned, "the loss is calculated after the action is obtained and the label is compared", is a supervised learning paradigm. We train the goal prior model to minimize the prediction loss betweem the output action label and the ground truth label from data.
